pkg/printers/internalversion/printers.go

```diff
@@ -827,11 +827,20 @@ func getServiceExternalIP(svc *api.Service, wide bool) string {
 		lbIps := loadBalancerStatusStringer(svc.Status.LoadBalancer, wide)
 		if len(svc.Spec.ExternalIPs) > 0 {
 			results := []string{}
+			mresults := make(map[string]bool)
```

Comment:
@imcsk8 could be fancy and use the optimization documented in https://stackoverflow.com/questions/37320287/maptstruct-and-maptbool-in-golang which seems to be in-vogue, eg instead of bool (since you don't care about the map value) use struct{} which apparently saves memory.  But given the limited number of items in the map, and the fact that it's short-lived, may not be worth it.